Clicker/src/js/components/Clicker/Clicker.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import Button from '../Button/Button.jsx';
+
+import './Clicker.scss';
+
+const Clicker = () => {
+
+    const [number, setNumber] = useState(0);
+
+    return (
+        <div className='clicker'>
+            <div className='clicker-result'>{number}</div>
+            <div className='clicker-buttons'>
+                <Button children='' className='increment-button' disabled={false} onClick={()=>setNumber(number+1)} />
```

Comment:
обычно prop `children` мы не используем напрямую, а передаем в качестве child element'ов внутрь компонента, так код выглядит более читабельным
```suggestion
                <Button className='increment-button' disabled={false} onClick={()=>setNumber(number+1)} />
                  {element}
                </Button>
```

в твоем случае, `children`  тут вообще не надо, можно не передавать просто

---

чтобы не передавать постоянно `disabled={false}`, можно в компонете присваивать дефолтное значение
```
const Button = ({ children, onClick, className, disabled = false, ...attrs }) => {

Reply:
Ок. Понял